helios-services/src/main/java/com/spotify/helios/rollingupdate/DeploymentGroupEventFactory.java

```diff
@@ -37,51 +36,68 @@
     HOSTS_CHANGED
   }
 
-  public Map<String, Object> rollingUpdateTaskSucceeded(final DeploymentGroup deploymentGroup,
-                                                        final RolloutTask task) {
+  private Map<String, Object> createEvent(final String eventType,
```

Comment:
For what objects?

A DeploymentGroupEvent class? Probably. But I'd rather do that in a separate commit.

Or specific classes for specific events?